config/locales/translations/zendesk_apps_support.yml

```diff
@@ -204,3 +204,19 @@ parts:
       key: "txt.apps.admin.error.app_build.invalid_type_parameter.other"
       title: "App builder job: multiple invalid parameter types error"
       value: "%{invalid_types} are invalid parameter types."
+  - translation:
+      key: "txt.apps.admin.error.app_build.unacceptable_boolean"
+      title: "App builder job: this value needs to be either true or false, but something else was passed in"
+      value: "%{field} must be a Boolean value, got %{value}"
+  - translation:
+      key: "txt.apps.admin.error.app_build.invalid_no_template"
+      title: "App builder job: `noTemplate` needs to be true, false or a valid array, but something else was passed in"
+      value: "noTemplate must be set to true, false, or an array of valid locations."
+  - translation:
+      key: "txt.apps.admin.error.app_build.duplicate_reference"
+      title: "App builder job: a key was included twice in the supplied manifest file, but it is only allowed once"
+      value: "Duplicate reference in manifest: %{key}."
+  - translation:
+      key: "txt.apps.admin.error.app_build.duplicate_reference_values"
+      title: "This sentence follows txt.apps.admin.error.app_build.duplicate_reference. The values are included to help find the problem."
+      value: "Initially set to %{original}, attempted overwrite to %{attempted}."
```

Comment:
Got it, then can we add quotes or something like Initially set to value "whatever", attempted overwrite to value "whatever"? That would avoid a lot of issues it would make sense